.github/workflows/test.yml

```diff
@@ -13,10 +13,10 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v2
 
-      - name: Set Up Python 3.8
+      - name: Set Up Python 3.10
         uses: actions/setup-python@v2
         with:
-          python-version: 3.8
+          python-version: "3.10"
```

Comment:
Ensure that the specified Python version is compatible with your project's dependencies. If your project supports multiple Python versions, consider testing against all of them.